intelmpi/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv, lib, gfortran, fetchurl, makeWrapper, rpmextract, which
+# Runtime dependencies
+, openssh
+}:
+stdenv.mkDerivation rec {
+    pname = "intelmpi";
+    version = "2019.7.217";
```

Comment:
Is that the latest version?

Reply:
No, it is not. It is just the version that works for MRCC. The Intel toolings have changed significantly in 2021. There is this HPC OneAPI" thing now, which includes the compilers, MPI and some analysis programs. 

In principle I think it would be very nice to package those. MRCC is nearly 20 times slower when building the minimal version with gfortran instead of ifort and MultiWFN does not build at all. But building a stdenv around the intel tools has been attempted and does not look easy (https://github.com/NixOS/nixpkgs/issues/32434).

But here in this case it would be a very different derivation. We could also just drop IntelMPI for now. It actually runs much better in the OMP parallel version and I had never any real use for the MPI parallel version.